cmake/add_project_pytorch.cmake

```diff
@@ -37,9 +37,9 @@ if( VIAME_ENABLE_PYTORCH-PYSOT )
   set( PYTORCH_LIBS_TO_BUILD ${PYTORCH_LIBS_TO_BUILD} pysot )
 endif()
 
-if( VIAME_ENABLE_PYTORCH-NETHARN )
-  set( PYTORCH_LIBS_TO_BUILD ${PYTORCH_LIBS_TO_BUILD} netharn )
-endif()
+#if( VIAME_ENABLE_PYTORCH-NETHARN )
+#  set( PYTORCH_LIBS_TO_BUILD ${PYTORCH_LIBS_TO_BUILD} netharn )
+#endif()
```

Comment:
Would recommend removing dead code (and also removing the subpackage in packages/netharn if its now installed via above pip call)

Reply:
Fixed.